src/AppKit/NSBezierPath.cs

```diff
@@ -110,5 +110,15 @@ public unsafe void AppendPathWithGlyphs (uint[] glyphs, NSFont font)
 				_AppendPathWithGlyphs ((IntPtr)ptr, glyphs.Length, font);
 		}
 
+		public unsafe void AppendBezierPathWithCGGlyphs (uint[] glyphs, NSFont font)
```

Comment:
I missed some deprecations from the diff. 

The other one was calling

		[Export ("appendBezierPathWithGlyphs:count:inFont:"), Internal]

which is deprecated

the new one is calling

		[Export ("appendBezierPathWithCGGlyphs:count:inFont:")]

which is 10.13